src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -1348,14 +1348,10 @@ private Set<AnswerPost> createBasicAnswersThatResolves(Post post, String userPre
         return answerPosts;
     }
 
-    public List<Course> createMultipleCoursesWithAllExercisesAndLectures(String userPrefix, int numberOfCoursesWithExercises, int numberOfCoursesWithLectures,
-            int numberOfTutorParticipations) throws Exception {
+    public List<Course> createMultipleCoursesWithAllExercisesAndLectures(String userPrefix, int numberOfCourses, int numberOfTutorParticipations) throws Exception {
```

Comment:
since `createCoursesWithExercisesAndLecturesAndLectureUnits` creates two courses already, `numberOfCourses` is misleading